### DISCUSSION TITLE
Can other users complete icon requests?

### DISCUSSION THREAD

**Discussion by colin-heffernan** [Arcticons-Team/Arcticons#551] [Q&A] [answered]
If I'm bored and have free time, can I try to complete some icon requests sent in by users unable to create icons themselves?

**Comment by Aman9das** [comment, marked as answer]
Yes, indeed!

**Reply by colin-heffernan** [reply]
Sweet! I might just do that, assuming I get that free time.

**Reply by Donnnno** [reply]
Just follow the guidelines and your icons will be added to the pack!
